components/camera/videosource/webcam.go

```diff
@@ -386,13 +389,19 @@ func (c *monitoredWebcam) isCameraConnected() (bool, error) {
 	if c.underlyingSource == nil {
 		return true, errors.New("no configured camera")
 	}
-	props, err := gostream.PropertiesFromMediaSource[image.Image, prop.Video](c.underlyingSource)
+	d, err := gostream.DriverFromMediaSource[image.Image, prop.Video](c.underlyingSource)
 	if err != nil {
-		return true, errors.Wrap(err, "cannot get properties from media source")
+		return true, errors.Wrap(err, "cannot get driver from media source")
 	}
-	// github.com/pion/mediadevices connects to the OS to get the props for a driver. On disconnect props will be empty.
+
 	// TODO(RSDK-1959): this only works for linux
```

Comment:
For Mac OS X this is still a no-op just as before. It should still trivially return as always connected.